src/state/migrations/0.ts

```diff
@@ -2,16 +2,19 @@ import { PersistedState } from 'redux-persist';
 
 import { parseStorageItem } from './utils';
 
+export type V0State = PersistedState & { tradingView: { chartConfig?: object } };
 /**
  * First migration (as an example), moving over tradingview chart configs
  *
  */
-export function migration0(state: PersistedState) {
+export function migration0(state: PersistedState | undefined): V0State {
```

Comment:
could we name these (and files) something more descriptive? as far as i can tell this naming convention isn't required.

Reply:
it is standard to name these files according to the order they are run: https://github.com/search?q=redux-persist+migration1&type=code